.github/workflows/fake-bpy-module-ci.yml

```diff
@@ -9,9 +9,27 @@ on:
   pull_request:
 
 jobs:
+  set_versions:
+    name: Set build versions
+    runs-on: ubuntu-latest
+    outputs:
+      module_version: ${{ steps.set_module_version.outputs.module_version }}
+      file_version: ${{ steps.set_file_version.outputs.file_version }}
+    steps:
+      # Use ISO 8601 date (in UTC) for release
+      - name: Set module version
+        id: set_module_version
+        run: echo ::set-output name=module_version::$(date -u +%Y%m%d)
+
+      # Use ISO 8601 timestamps (in UTC) for output/file version
+      - name: Set file version
+        id: set_file_version
+        run: echo ::set-output name=file_version::$(date -u +%Y%m%dT%H%M%SZ)
```

Comment:
I think the output name is a bit long.
Why do you need `%H%M%S` as well?

Reply:
@nutti Because PRs, commits on master and potentially other CI-triggers can run at the same time:
> I chose a second-exact resolution timestamp for the files, as we might have multiple builds per day. While all those artifact filenames have a timestamp as a version, I did not (as requested) modify the RELEASE_VERSION.

Reply:
Thanks, understood.